components/patient-data/ui/src/main/resources/PhenoTips/PatientConsent.xml

```diff
@@ -796,9 +795,6 @@
       (% class='view-title-required-not-granted hidden' %)(((
         {{warning}}$services.localization.render('PhenoTips.PatientConsent_required_not_granted'){{/warning}}
       )))
-      (% class='view-title-none-granted hidden' %)(((
-        {{warning}}$services.localization.render('PhenoTips.PatientConsent_none_granted'){{/warning}}
```

Comment:
I think we should also remove the `PhenoTips.PatientConsent_none_granted` string from app.properties